source/gridiron/index.haml

```diff
@@ -29,38 +29,87 @@ description: Aptible Gridiron is your team's first security hire. Design, operat
 
 = partial "partials/gridiron2/sticky-nav-gridiron"
 
-%section.light-blue-angle-divider
-  .gridiron2__content-block
-    %h1 Hundreds of startups use Aptible to streamline audit prep and reduce the burden of vendor security assessments
-    .text-center
-      %img{ src: '/images/gridiron2/logo-row.svg', style: 'padding: 30px 0' }
-
-    .gridiron2__testimonial
-      = partial "partials/gridiron2/testimonial-snaps"
-
 %section
   .gridiron2__content-block
-    %h1 Complete information security audit prep, built for startups
+    %h1 Complete information security audit prep for startups in a fraction of the time
     .row
       .third
         .gridiron2__icon
           %img{ src: '/images/gridiron2/icon-speed.svg' }
-        %h2 Fast
-        %p Fast, as in you’re up-and-running in hours instead of weeks or months. Get audit-ready and get back to growing your business.
+        %h2 Acceleration through guidance
+        %p Guidance that gets you up and running in hours instead of weeks or months. Get audit-ready and get back to focusing on your business.
 
       .third
         .gridiron2__icon
           %img{ src: '/images/gridiron2/icon-startups.svg' }
         %h2 Designed for Startups
-        %p Instructions, policies, team training, and security management features and services designed specifically for startups looking to open new sales channels or streamline the enterprise sales cycle. 
+        %p Instructions, policies, team training, and security management features and services designed specifically for startups looking to build customer trust, open new sales channels, or streamline the enterprise sales cycle.
 
       .third
         .gridiron2__icon
           %img{ src: '/images/gridiron2/icon-pricing.svg' }
         %h2 Cost-effective
         %p The Gridiron platform delivers more comprehensive audit prep at a fraction of the cost of alternatives. It's priced at about 1/5th the cost of hiring a security lead, and less than most consultants.
+    .row
+      .third
+        .gridiron2__icon
+          %img{ src: '/images/gridiron2/icon-guide.svg' }
+        %h2 A guide to walk you through
+        %p Aptible provides compliance lawyers who give real-time advice, guidance, and support as you design, operate, and audit your security management system.
```

Comment:
Can we change this from "compliance lawyers" to "compliance and data protection advisors"